contribs/gmf/less/desktop.less

```diff
@@ -20,6 +20,10 @@
 @toolbar-width: 4.5rem;
 @border-color: darken(@brand-primary, @standard-variation);
 @search-width: 8 * @map-tools-size;
+@font-size-base: 13px;
+@padding-base-vertical: 5px;
+@padding-base-horizontal: 10px;
+@form-group-margin-bottom: 10px;
```

Comment:
It will be probably not so much work to change those variables compared to the other required modifications when upgrading.